packages/headless/src/controllers/core/search-parameter-manager/headless-core-search-parameter-manager.ts

```diff
@@ -282,3 +301,25 @@ function getAutomaticFacets(state: Partial<SearchParametersState>) {
 
   return Object.keys(af).length ? {af} : {};
 }
+
+function loadSearchParameterManagerReducers(
+  engine: CoreEngine
+): engine is CoreEngine<SearchParameterManager> {
+  engine.addReducers({
```

Comment:
Reminder: this change will render the code sample in this documentation page kind of pointless https://docs.coveo.com/en/atomic/latest/usage/headless-through-atomic/, so it should be updated.

(I know you're aware of this documentation, but it's easy to forget about those things 😅)

Reply:
Should add smthg smwhere to not forget ;)